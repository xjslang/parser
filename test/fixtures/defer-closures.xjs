function testClosures() {
  console.log('function start')
  
  for (let i = 0; i < 3; i++) {
    defer {
      console.log('defer iteration:', i)
    }
  }
  
  console.log('function end')
}

testClosures()
