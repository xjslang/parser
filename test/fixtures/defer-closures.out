function start
function end
defer iteration: 2
defer iteration: 1
defer iteration: 0
